src/js/view/preview.js

```diff
@@ -63,6 +87,17 @@ Object.assign(Preview.prototype, {
     },
     element: function() {
         return this.el;
+    },
+    destroy: function() {
+        if (this.hasZoomThumbnail) {
+            this.el.classList.remove('jw-ab-zoom-thumbnail');
+            this.model.off('change:state', this.destroy, this);
+            this.model.off('change:viewable', this.pauseZoom, this);
```

Comment:
This will remove all model listeners for this instance.
`this.model.off(null, null, this);`